macro/src/main/scala-3.x/monocle/macros/GenPrism.scala

```diff
@@ -1,9 +1,18 @@
 package monocle.macros
 
-import monocle.Focus
-import monocle.Focus._
+import monocle.Prism
+import scala.quoted.{Type, Expr, Quotes}
 
 object GenPrism {
```

Comment:
Sorry I should have added a comment. I inlined the macro because of the regression in scala 3 rc1. Hopefully we can rollback to focus implementation in rc2.

It is important to build against the latest version to discover all the differences and publish an artifact for downstream projects.

Reply:
You're right. I completely misunderstood the problem. Rolling it back.